### ISSUE TITLE
project: rename to laxar-uikit

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar-uikit#121]
Like with LaxarJS patterns, the LaxarJS UiKit repository should be  should be renamed to `laxar-uikit` (dash, not underscore).

**Comment by x1B** [comment]
Marked _breaking_ because this affects the RequireJS name (mandating a `map` configuration entry).

**Comment by x1B** [comment]
Implemented on `master` (`v1.0.0`).

### Breaking Change

The artifact `laxar_uikit` and its main git repository have been renamed to `laxar-uikit`. 
For existing widgets, controls and applications to keep working, adjust your project's RequireJS configuration:

Rename `laxar_uikit` to `laxar-uikit` in the package reference:
```js
   packages: [
      // ...
      {
         name: 'laxar-uikit',
         location: '../includes/lib/laxar-uikit',
         main: 'laxar-uikit'
      }
   },
// ...
```

Add a mapping so that existing references to `laxar_uikit` can be resolved:
```js
   map: {
      '*': {
         // ...
         'laxar_uikit': 'laxar-uikit',
         // LaxarJS patterns was also renamed
         'laxar_patterns': 'laxar-patterns'
      }
   },
   // ...
```
